python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -26,8 +26,9 @@
 
 pytest.importorskip("dbt.version", "1.6")
 
-dagster_dbt_translator_with_checks = DagsterDbtTranslator(
-    settings=DagsterDbtTranslatorSettings(enable_asset_checks=True)
+
+dagster_dbt_translator_without_checks = DagsterDbtTranslator(
```

Comment:
[1]

I think we should keep this object for tests.

The tests that I would add here are:
- We test that a default `DagsterDbtTranslator` without settings passed in results in asset checks being enabled
- Introduce a separate `dagster_dbt_translator_without_checks` and test that asset checks are not enabled when that object is passed. 
- We keep the existing tests as they are. This way, we're not relying on the implicit behavior to do our testing for us.